diagrams/request_uri_mode_post.md

```diff
@@ -0,0 +1,50 @@
+```plantuml
+@startuml
+
+autonumber
+
+box "Wallet"
+participant "Metadata" as wm
+participant "Authorization Endpoint" as w
+end box
+
+participant "User Agent" as u
+
+box "Verifier"
+participant "Frontend" as r
+participant "Request Endpoint" as rp
+participant "Response Endpoint" as rb
+end box
+
+u --> r : use
+activate r
+
+r --> u: authorization request\n(client_id, request{request_uri, request_uri_mode=POST, state})
+deactivate r
+u --> w: authorization request\n(client_id, request{request_uri, request_uri_mode=POST, state})
+activate w
+w --> w: check authorization request signature
+w --> w: check on trustworthiness of Verifier (approach on trust mechanism)
```

Comment:
I think these steps are not needed for this feature. We should not mandate signing the initial request. These steps are performed by the trust framework. The remaining approach looks good to me, i.e., sending SELECTED wallet capabilities which should be a subset of  obtained client metadata.